docs/development/getting_started.rst

```diff
@@ -0,0 +1,66 @@
+===============
+Getting started
+===============
+
+
+.. contents:: :local:
+
+
+General
+-------
+
+- the minimal python version is Python 3.6.0, don't break it,
+- VmShepherd is a asynchronous app. Keep in mind that every IO operation should be implemented using asyncio,
```

Comment:
I think this sentence is not ended, for me is missing a part:
using ayncio because .... <-[link] or something

Reply:
This is getting started, not design rationale

Reply:
But of course, the point is right and relevant docs need to be completed.

Reply:
okay, sounds fair ;)